core/src/main/java/com/datastax/oss/driver/internal/core/protocol/CompressorSubstitutions.java

```diff
@@ -83,11 +82,9 @@ public static Compressor<ByteBuf> newInstance(String name, DriverContext context
   }
 
   @TargetClass(value = Lz4Compressor.class, onlyWith = Lz4Missing.class)
-  @Delete
```

Comment:
The original PR deletes the whole class, not just the annotation. What is the difference?

Reply:
I think it would make sense to keep what did and remove the whole class for the 2 `@Delete` substitutions.

Reply:
Apologies, this was a mistake on my part... that's what I get for not reading carefully.  Fixing now.

Reply:
Done